components/Navbar.tsx

```
@@ -0,0 +1,20 @@
+import { TextRipple } from "@/components/ui/text-ripple";
+
+export default function Navbar() {
+  return (
+    <nav className="mx-6 mt-5 flex h-24 items-center justify-between rounded-xl bg-primary px-16 font-instrument-serif text-secondary">
+      <TextRipple className="text-4xl">The Incubator.</TextRipple>
+        <div className="flex gap-6">
+          <button className="rounded-lg px-6 py-2 font-sf-pro text-lg font-medium text-secondary underline decoration-wavy decoration-1 underline-offset-4 transition-all duration-100 hover:scale-x-110 hover:scale-y-95 hover:decoration-zinc-400">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)